packages/common/aws.js

```diff
@@ -594,5 +596,7 @@ exports.setGranuleStatus = async (
 ) => {
   const key = exports.getGranuleS3Params(granuleId, stack, bucket);
   const executionArn = exports.getExecutionArn(stateMachineArn, executionName);
-  await exports.s3().putObject(bucket, key, '', null, { executionArn, status }).promise();
+  const params = { Bucket: bucket, Key: key };
+  params.Metadata = { executionArn, status };
+  await exports.s3().putObject(params).promise();
```

Comment:
I've filed a ticket to take a look at this functionality:  https://bugs.earthdata.nasa.gov/browse/CUMULUS-420